hack/test-cmd.sh

```diff
@@ -132,67 +132,68 @@ for version in "${kube_api_versions[@]}"; do
 
   kube::log::status "Testing kubectl(${version}:pods)"
   kubectl get pods "${kube_flags[@]}"
-  kubectl create -f examples/guestbook/redis-master.json "${kube_flags[@]}"
+  kubectl create -f examples/limitrange/valid-pod.json "${kube_flags[@]}"
   kubectl get pods "${kube_flags[@]}"
-  kubectl get pod redis-master "${kube_flags[@]}"
-  [ "$(kubectl get pod redis-master -o template --output-version=v1beta1 -t '{{ .id }}' "${kube_flags[@]}")" == "redis-master" ]
-  output_pod=$(kubectl get pod redis-master -o yaml --output-version=v1beta1 "${kube_flags[@]}")
-  kubectl delete pod redis-master "${kube_flags[@]}"
+  kubectl get pod valid-pod "${kube_flags[@]}"
+  [ "$(kubectl get pod valid-pod -o template --output-version=v1beta1 -t '{{ .id }}' "${kube_flags[@]}")" == "valid-pod" ]
+  output_pod=$(kubectl get pod valid-pod -o yaml --output-version=v1beta1 "${kube_flags[@]}")
+  kubectl delete pod valid-pod "${kube_flags[@]}"
   before="$(kubectl get pods -o template -t "{{ len .items }}" "${kube_flags[@]}")"
   echo "${output_pod}" | kubectl create -f - "${kube_flags[@]}"
   after="$(kubectl get pods -o template -t "{{ len .items }}" "${kube_flags[@]}")"
   [ "$((${after} - ${before}))" -eq 1 ]
-  kubectl get pods -o yaml --output-version=v1beta1 "${kube_flags[@]}" | grep -q "id: redis-master"
-  kubectl describe pod redis-master "${kube_flags[@]}" | grep -q 'Name:.*redis-master'
-  kubectl delete -f examples/guestbook/redis-master.json "${kube_flags[@]}"
-  kubectl delete pods -l name=redis-master "${kube_flags[@]}"
-  [ ! $(kubectl get pods "${kube_flags[@]}" -lname=redis-master | grep -q 'redis-master') ]
-  kubectl create -f examples/guestbook/redis-master.json "${kube_flags[@]}"
-  kubectl get pods "${kube_flags[@]}" -lname=redis-master | grep -q 'redis-master'
+  kubectl get pods -o yaml --output-version=v1beta1 "${kube_flags[@]}" | grep -q "id: valid-pod"
+  kubectl describe pod valid-pod "${kube_flags[@]}" | grep -q 'Name:.*valid-pod'
+  kubectl delete -f examples/limitrange/valid-pod.json "${kube_flags[@]}"
+  
+  kubectl delete pods -l name=valid-pod "${kube_flags[@]}"
+  [ ! $(kubectl get pods "${kube_flags[@]}" -lname=valid-pod | grep -q 'valid-pod') ]
+  kubectl create -f examples/limitrange/valid-pod.json "${kube_flags[@]}"
+  kubectl get pods "${kube_flags[@]}" -lname=valid-pod | grep -q 'valid-pod'
   [ ! $(kubectl delete pods "${kube_flags[@]}" ) ]
-  kubectl get pods "${kube_flags[@]}" -lname=redis-master | grep -q 'redis-master'
-  [ ! $(kubectl delete pods --all pods -l name=redis-master "${kube_flags[@]}" ) ]   # not --all and label selector together
+  kubectl get pods "${kube_flags[@]}" -lname=valid-pod | grep -q 'valid-pod'
+  [ ! $(kubectl delete pods --all pods -l name=valid-pod "${kube_flags[@]}" ) ]   # not --all and label selector together
```

Comment:
Latest conflict was here due to https://github.com/GoogleCloudPlatform/kubernetes/commit/29aae753299981140550397105a3b61c0ea6d38e